### ISSUE TITLE
F-Droid build failed

### ISSUE THREAD

**Issue by linsui** [dstark5/Openlib#129]
https://gitlab.com/fdroid/checkupdates-bot-fdroiddata/-/jobs/8252325792

https://gitlab.com/fdroid/fdroiddata/-/merge_requests/16283

I thought this is because we enabled shrinkResources to remove the nonfree libs from flutter. Could you please take a look? Thanks!

**Comment by dstark5** [comment]
Hi @linsui,
we need to have "shrinkResources = false" as the app uses a epub reader which requires this without it the build will fail

**Comment by linsui** [comment]
Maybe this can be solved by proper proguard rules?

**Comment by dstark5** [comment]
I don't know but will look on it, so without shrink the app build won't be available in fdroid?

**Comment by linsui** [comment]
Yep, see https://gitlab.com/fdroid/fdroiddata/-/issues/2949 for more details. :)

**Comment by dstark5** [comment]
@linsui Thank you will look into it
